.travis.yml

```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 5.6
+  - 7.0
```

Comment:
@VaclavSir -> [DameJidlo.cz member] ... ještě pořád! @legendik fix it! :trollface: